templates/ai/consulting.html

```diff
@@ -0,0 +1,349 @@
+        alt="" 
+        width="335px"
+      />
+    </div>
+    <div class="col-6">
+      <h4>5 - Operation and Feedback</h4>
+      <p>
+        This may include UI
+        development, technical documentation, and hands-on knowledge transfer
+        between development and operations teams to ensure they can operate the
+        solution. It is important to discuss production deployment options – for
+        example, a dark launch or an integration with a prototypical business
+        application. During this step we’ll elicit feedback from key
+        stakeholders, covering the baseline architecture and model.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <hr class="p-separator"/>
+  </div>
+  <div class="row">
+    <p>
+      The specific steps for each phase will be adjusted based on your
+      particular needs.
+    </p>
+  </div>
+</section>
+
+<section class="p-strip--light is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-8">
+      <h2>Perfectly portable AI workloads</h2>
+      <h4 class="u-sv3">
+        Open and pluggable machine learning workflows for multi-cloud AI, ML and
+        DL.
+      </h4>
+      <p>
+        In the multi-cloud AI world, where AI practitioners leverage multiple
+        environments, portability requirements surface rather quickly. We
+        consider each environment to be different and in need of the ability to
+        define a repeatable machine learning stack.
+      </p>
+      <p>
+        The full AI/ML workflow starts with the developer workstation, for rapid
+        iteration of algorithms and analytic techniques. Training at scale takes
+        place on the rack or on GPGPU-accelerated instances on the cloud.
+        Resulting models are then delivered to the point of decision, whether
+        that’s on cloud or IoT at the edge.
+      </p>
+      <p>
+        Working with Canonical and all the leading clouds, Kubeflow on Ubuntu
+        provides infrastructure and operations that span that full range, and
+        seamless portability of AI workloads.
+      </p>
+      <a class="p-button--neutral" href="/engage/deploy-ai-ml-model?utm_campaign=FY19_Cloud_K8_WBN_AIWebinar2">
+        Explore Kubeflow
+      </a>
+    </div>
+    <div class="col-4 prefix-1 u-vertically-center u-align--center u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}da5a7c9e-Kubeflow-Logo.svg" alt="" width="180px">
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <h2>Train your infrastructure engineers</h2>
+    <p>
+      Get more detailed information in our
+      <a
+        class="p-link--external"
+        href="{{ ASSET_SERVER_URL }}223eab1a-canonical-kubernetes-enterprise-2018-25-07.pdf"
+        >consulting datasheet</a
+      >
+    </p>
+  </div>
+
+  {% include "shared/pricing/_kubernetes-consulting-add-ons.html" %}
+
+  <div class="row">
+    <p>
+      <a href="/ai/contact-us?product=ai-index"
```

Comment:
there is a contact us in the include, I think remove this and from the copy doc too.